README.md

```diff
@@ -5,111 +5,204 @@
 
-[//]: # (TODO: Add examples)
+1. [Create a Zoom meeting](https://github.com/module-ballerinax-zoom.meetings/tree/main/examples/create-new-meeting) – Creates a new Zoom meeting using the API. 
 
-## Build from the source
+2. [List scheduled meetings](https://github.com/module-ballerinax-zoom.meetings/tree/main/examples/list-all-meetings) – Displays the list of meetings scheduled under a specified Zoom user account. 
 
 ### Setting up the prerequisites
-
 1. Download and install Java SE Development Kit (JDK) version 21. You can download it from either of the following sources:
-
     * [Oracle JDK](https://www.oracle.com/java/technologies/downloads/)
     * [OpenJDK](https://adoptium.net/)
-
    > **Note:** After installation, remember to set the `JAVA_HOME` environment variable to the directory where JDK was installed.
-
 2. Download and install [Ballerina Swan Lake](https://ballerina.io/).
-
 3. Download and install [Docker](https://www.docker.com/get-started).
-
    > **Note**: Ensure that the Docker daemon is running before executing any tests.
-
 4. Export Github Personal access token with read package permissions as follows,
-
     ```bash
     export packageUser=<Username>
     export packagePAT=<Personal access token>
     ```
-
 ### Build options
-
 Execute the commands below to build from the source.
-
 1. To build the package:
-
    ```bash
    ./gradlew clean build
    ```
-
 2. To run the tests:
-
    ```bash
    ./gradlew clean test
    ```
-
 3. To build the without the tests:
-
    ```bash
    ./gradlew clean build -x test
    ```
-
 4. To run tests against different environments:
-
    ```bash
    ./gradlew clean test -Pgroups=<Comma separated groups/test cases>
    ```
-
 5. To debug the package with a remote debugger:
-
    ```bash
    ./gradlew clean build -Pdebug=<port>
    ```
-
 6. To debug with the Ballerina language:
-
    ```bash
    ./gradlew clean build -PbalJavaDebug=<port>
    ```
-
 7. Publish the generated artifacts to the local Ballerina Central repository:
-
     ```bash
     ./gradlew clean build -PpublishToLocalCentral=true
     ```
-
 8. Publish the generated artifacts to the Ballerina Central repository:
-
    ```bash
    ./gradlew clean build -PpublishToCentral=true
    ```
-
 ## Contribute to Ballerina
-
 As an open-source project, Ballerina welcomes contributions from the community.
-
 For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
-
 ## Code of conduct
-
 All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
-
 ## Useful links
-
 * For more information go to the [`zoom.meetings` package](https://central.ballerina.io/ballerinax/zoom.meetings/latest).
 * For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).
 * Chat live with us via our [Discord server](https://discord.gg/ballerinalang).
```

Comment:
Let's not remove the empty lines here. This will cause the file to render incorrectly.